### PR TITLE
Add Zoho Mail

### PR DESCRIPTION
**Pull request by jaredkotoff**

### CODE REVIEW

File: uncompressed/zoho/webview.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  const getMessages = () => {
+    const unreadMail = $(".zm_urd").not(".hidden").length;
+
+    Franz.setBadge(unreadMail);
+  }
+
+  Franz.loop(getMessages);
+
+}
+};
```
Comment (vantezzen) [inline uncompressed/zoho/webview.js]: Excess bracket - please move this.

**Comment by jaredkotoff** [comment]
@vantezzen can you let me know what I might need to do to fix this?  
`yarn verify-all` doesn't work at all on my machine

**Comment by vantezzen** [comment]
I am sorry for the inconvenience our system caused - we try to make it as easy as possible to integrate new recipes.
I have copied your files from this PR into the master branch and now everything seems to work. The reason the check on your PR failed is probably because you completely deleted the asana archive, which makes the asana recipe invalid.

I've now merged your files into master via 21bebcc so I will close this PR as your changes merged. Again sorry for the inconvenience.

Our of interest, what OS are you trying to package the recipe on?

**Comment by jaredkotoff** [comment]
Yea, windows. Although, recently learned of running VSCode in WSL going to try to start using that: https://code.visualstudio.com/docs/remote/wsl

**Pull request closed without merging**